docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
+    The problem is xhyve driver is not able to clean up vmnet when remove a VM.
+
```

Comment:
The problem is likely to be that the xhyve driver is not able to clean up vmnet when a VM is removed.